blocks/shared-styles/_children/byline/index.scss

```diff
@@ -0,0 +1,31 @@
+.byline {
+  $self: &;
+  color: #191919;
+  font-size: 1.25rem;
+  line-height: 2.5rem;
+
+  &--list {
+    display: inline;
+    font-size: 0.875rem;
+    line-height: 1rem;
+
+    #{$self}__by {
+      color: #3b3b3b;
```

Comment:
not sure if this is new but can we use arc variables for colors if possible? I believe they're in the news theme css

Reply:
This color value isn't in news-theme-css.. We could create a ticket for this - I just brought over the existing styled component logic

Reply:
ok no worries, thought this was new